mozilla_schema_generator/utils.py

```diff
@@ -15,6 +15,9 @@ def _get(_dict: dict, key: Tuple[str]) -> Any:
     Retrieved the nested `key` from a dict
     """
     if key:
+        if key[0] not in _dict:
```

Comment:
Can you describe why the behavior of `_get` needs to change?

Reply:
I added a comment in the code. Basically, before `_get` would throw an exception for accessing non-existend fields

Reply:
Hmm. That was by design, so if an unexpected field was trying to be retrieved, we would know with a hard fail. If it's failing somewhere it sounds like that may be a bug.